test/unit/handle-runtime-problems-test.coffee

```diff
@@ -62,7 +62,7 @@ describe('handleRuntimeProblems()', ->
     beforeEach((done) ->
       prepareData(apiDescriptionDocument, filename, (err, data) ->
         error = handleRuntimeProblems(data)
```

Comment:
I should stop doing before thinking. Happens to me a lot lately :neutral_face: (this is genuine neutral face, not Dredd's smile)